### PR TITLE
fix broken links[HSTN-899]

### PR DESCRIPTION
**Pull request by hngerebara**
fix broken links

### CODE REVIEW

File: examples/influxdb-ami/README.md [hunk 1]
```diff
@@ -24,9 +24,8 @@ To build the InfluxDB AMI:
 1. To build an Amazon Linux AMI for InfluxDB Enterprise: `packer build -only=amazon-linux-ami influxdb.json`.
 
 When the build finishes, it will output the IDs of the new AMIs. To see how to deploy this AMI, check out the 
-[influxdb-cluster-simple](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-cluster-simple) and
-[influxdb-multi-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-multi-cluster)
```
Comment (brikis98) [inline examples/influxdb-ami/README.md]: Why remove the second link? It should prob go to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/tick-multi-cluster

File: examples/influxdb-ami/README.md [hunk 2]
```diff
@@ -75,6 +74,4 @@ local checkout of the Module, uploaded from your own computer via the `file` pro
 
 The Packer template in this example folder can build not only AMIs, but also Docker images for local testing. This is
 convenient for testing out the various scripts in the `modules` folder without having to wait for an AMI to build and
-a bunch of EC2 Instances to boot up. See the [influxdb-docker
```
Comment (brikis98) [inline examples/influxdb-ami/README.md]: Should link to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/local-mocks

File: examples/influxdb-cluster-simple/README.md [hunk 3]
```diff
@@ -14,9 +14,7 @@ You will need to create an [Amazon Machine Image (AMI)](http://docs.aws.amazon.c
 that has InfluxDB installed, which you can do using the [influxdb-ami 
 example](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/influxdb-ami)). 
 
-To see an example of InfluxDB deployed across separate clusters, see the [influxdb-multi-cluster
-example](https://github.com/gruntwork-io/terraform-aws-influx/blob/master/examples/influxdb-multi-cluster). For
-more info on how the InfluxDB cluster works, check out the
```
Comment (brikis98) [inline examples/influxdb-cluster-simple/README.md]: Should link to https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples/tick-multi-cluster

File: examples/tick-single-cluster/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # TICK Single Cluster Example
 
 This folder shows an example of Terraform code that uses the
-[tick-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/tick-cluster) 
+tick-cluster
```
Comment (brikis98) [inline examples/tick-single-cluster/README.md]: Should say and link to [influxdb-cluster](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/influxdb-cluster)

File: modules/influxdb-security-group-rules/README.md [hunk 5]
```diff
@@ -6,8 +6,8 @@ These rules are defined in a separate module so that you can add them to any exi
 
 ## Quick start
 
-Let's say you want to deploy influx using the [influx-cluster 
-module](https://github.com/gruntwork-io/terraform-aws-influx/blob/master/modules/influx-cluster): 
+Let's say you want to deploy influx using the [influxdb-cluster 
+module](/modules/influxdb-cluster/README.md):
```
Comment (brikis98) [inline modules/influxdb-security-group-rules/README.md]: Needs to be absolute URL

File: modules/install-influxdb/README.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
 This folder contains a script for installing InfluxDB and its dependencies. Use this script to create an
 InfluxDB [Amazon Machine Image (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) that can be
 deployed in [AWS](https://aws.amazon.com/) across an Auto Scaling Group using the [influxdb-cluster
-module](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/influxdb-cluster).
+module](/modules/influxdb-cluster/README.md).
```
Comment (brikis98) [inline modules/install-influxdb/README.md]: Same for all of these. This is an open source repo that goes in the Terraform Registry, so these must be absolute URLs

File: examples/influxdb-ami/README.md (hunk 1 above)
Comment (hngerebara) [reply examples/influxdb-ami/README.md]: okay. I removed it because of `influxdb-multi-cluster` doesn't exist

**Comment by hngerebara** [comment]
@brikis98 all done

**Pull request merged**